INFO - 2024-01-30 01:58:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-30 02:00:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-30 02:00:21 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bcpet_db', 3306, '', 0)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\Project (external)\BCPetHouse\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 D:\Project (external)\BCPetHouse\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 D:\Project (external)\BCPetHouse\app\Models\PetProfileModel.php(36): CodeIgniter\BaseModel->findAll()
#7 D:\Project (external)\BCPetHouse\app\Controllers\PetList.php(20): App\Models\PetProfileModel->getPetProfile()
#8 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\PetList->index()
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetList))
#10 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\Project (external)\BCPetHouse\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 D:\Project (external)\BCPetHouse\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 D:\Project (external)\BCPetHouse\app\Models\PetProfileModel.php(36): CodeIgniter\BaseModel->findAll()
#6 D:\Project (external)\BCPetHouse\app\Controllers\PetList.php(20): App\Models\PetProfileModel->getPetProfile()
#7 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\PetList->index()
#8 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetList))
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#12 {main}
CRITICAL - 2024-01-30 02:00:21 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_profile`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\PetProfileModel.php(36): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\PetList.php(20): App\Models\PetProfileModel->getPetProfile()
 7 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\PetList->index()
 8 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetList))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
INFO - 2024-01-30 02:00:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-30 02:00:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-30 02:00:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-30 02:01:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-30 02:01:25 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pe...', 0)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 D:\Project (external)\BCPetHouse\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Project (external)\BCPetHouse\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 D:\Project (external)\BCPetHouse\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\Project (external)\BCPetHouse\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 D:\Project (external)\BCPetHouse\app\Controllers\Ambulatoir.php(171): CodeIgniter\BaseModel->save(Array)
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
#10 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
#11 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#14 {main}
CRITICAL - 2024-01-30 02:01:25 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_profile` (`owner_name`, `name`, `age`, `address`, `phone`, `animal_type`, `race`, `fur_color`, `gender`, `created_at`, `updated_at`) VALUES (:owner_name:, :name:, :age:, :address:, :phone:, :animal_type:, :race:, :fur_color:, :gender:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Ambulatoir.php(171): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
 8 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
INFO - 2024-01-30 02:01:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-30 02:03:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-30 02:03:22 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pe...', 0)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 D:\Project (external)\BCPetHouse\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Project (external)\BCPetHouse\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 D:\Project (external)\BCPetHouse\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\Project (external)\BCPetHouse\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 D:\Project (external)\BCPetHouse\app\Controllers\Ambulatoir.php(171): CodeIgniter\BaseModel->save(Array)
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
#10 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
#11 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#14 {main}
CRITICAL - 2024-01-30 02:03:22 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_profile` (`owner_name`, `name`, `age`, `address`, `phone`, `animal_type`, `race`, `fur_color`, `gender`, `created_at`, `updated_at`) VALUES (:owner_name:, :name:, :age:, :address:, :phone:, :animal_type:, :race:, :fur_color:, :gender:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Ambulatoir.php(171): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
 8 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
INFO - 2024-01-30 02:03:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-30 02:03:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-30 02:03:29 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pe...', 0)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 D:\Project (external)\BCPetHouse\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Project (external)\BCPetHouse\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 D:\Project (external)\BCPetHouse\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\Project (external)\BCPetHouse\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 D:\Project (external)\BCPetHouse\app\Controllers\Ambulatoir.php(171): CodeIgniter\BaseModel->save(Array)
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
#10 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
#11 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#14 {main}
CRITICAL - 2024-01-30 02:03:29 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_profile` (`owner_name`, `name`, `age`, `address`, `phone`, `animal_type`, `race`, `fur_color`, `gender`, `created_at`, `updated_at`) VALUES (:owner_name:, :name:, :age:, :address:, :phone:, :animal_type:, :race:, :fur_color:, :gender:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Ambulatoir.php(171): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
 8 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
INFO - 2024-01-30 02:05:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-30 02:05:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-30 02:05:37 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pe...', 0)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 D:\Project (external)\BCPetHouse\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Project (external)\BCPetHouse\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 D:\Project (external)\BCPetHouse\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\Project (external)\BCPetHouse\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 D:\Project (external)\BCPetHouse\app\Controllers\Ambulatoir.php(171): CodeIgniter\BaseModel->save(Array)
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
#10 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
#11 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#14 {main}
CRITICAL - 2024-01-30 02:05:37 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_profile` (`owner_name`, `name`, `age`, `address`, `phone`, `animal_type`, `race`, `fur_color`, `gender`, `created_at`, `updated_at`) VALUES (:owner_name:, :name:, :age:, :address:, :phone:, :animal_type:, :race:, :fur_color:, :gender:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Ambulatoir.php(171): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
 8 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
INFO - 2024-01-30 02:09:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-30 02:09:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-30 02:09:07 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bcpet_db`.`ambulatoir`, CONSTRAINT `FK_ambulatoir_pet_profile` FOREIGN KEY (`pet_id`) REFERENCES `pet_profile` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION) in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `am...', 0)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `am...')
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `am...')
#3 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `am...', Array, false)
#4 D:\Project (external)\BCPetHouse\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Project (external)\BCPetHouse\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 D:\Project (external)\BCPetHouse\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\Project (external)\BCPetHouse\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 D:\Project (external)\BCPetHouse\app\Controllers\Ambulatoir.php(186): CodeIgniter\BaseModel->save(Array)
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
#10 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
#11 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#14 {main}
CRITICAL - 2024-01-30 02:09:07 --> Cannot add or update a child row: a foreign key constraint fails (`bcpet_db`.`ambulatoir`, CONSTRAINT `FK_ambulatoir_pet_profile` FOREIGN KEY (`pet_id`) REFERENCES `pet_profile` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ambulatoir` (`date_checkup`, `amnesa`, `status_present`, `clinical_finding`, `medication`, `created_at`, `updated_at`) VALUES (:date_checkup:, :amnesa:, :status_present:, :clinical_finding:, :medication:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Ambulatoir.php(186): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
 8 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
INFO - 2024-01-30 09:47:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-30 09:47:12 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bcpet_db`.`ambulatoir`, CONSTRAINT `FK_ambulatoir_pet_profile` FOREIGN KEY (`pet_id`) REFERENCES `pet_profile` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION) in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `am...', 0)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `am...')
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `am...')
#3 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `am...', Array, false)
#4 D:\Project (external)\BCPetHouse\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Project (external)\BCPetHouse\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 D:\Project (external)\BCPetHouse\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\Project (external)\BCPetHouse\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 D:\Project (external)\BCPetHouse\app\Controllers\Ambulatoir.php(186): CodeIgniter\BaseModel->save(Array)
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
#10 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
#11 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#14 {main}
CRITICAL - 2024-01-30 09:47:12 --> Cannot add or update a child row: a foreign key constraint fails (`bcpet_db`.`ambulatoir`, CONSTRAINT `FK_ambulatoir_pet_profile` FOREIGN KEY (`pet_id`) REFERENCES `pet_profile` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ambulatoir` (`date_checkup`, `amnesa`, `status_present`, `clinical_finding`, `medication`, `created_at`, `updated_at`) VALUES (:date_checkup:, :amnesa:, :status_present:, :clinical_finding:, :medication:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Ambulatoir.php(186): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
 8 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
INFO - 2024-01-30 09:55:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-30 10:14:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-30 10:14:11 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bcpet_db`.`ambulatoir`, CONSTRAINT `FK_ambulatoir_pet_profile` FOREIGN KEY (`pet_id`) REFERENCES `pet_profile` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION) in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `am...', 0)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `am...')
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `am...')
#3 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `am...', Array, false)
#4 D:\Project (external)\BCPetHouse\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Project (external)\BCPetHouse\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 D:\Project (external)\BCPetHouse\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\Project (external)\BCPetHouse\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 D:\Project (external)\BCPetHouse\app\Controllers\Ambulatoir.php(186): CodeIgniter\BaseModel->save(Array, true)
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
#10 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
#11 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#14 {main}
CRITICAL - 2024-01-30 10:14:11 --> Cannot add or update a child row: a foreign key constraint fails (`bcpet_db`.`ambulatoir`, CONSTRAINT `FK_ambulatoir_pet_profile` FOREIGN KEY (`pet_id`) REFERENCES `pet_profile` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ambulatoir` (`date_checkup`, `amnesa`, `status_present`, `clinical_finding`, `medication`, `created_at`, `updated_at`) VALUES (:date_checkup:, :amnesa:, :status_present:, :clinical_finding:, :medication:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Ambulatoir.php(186): CodeIgniter\BaseModel->save([...], true)
 7 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
 8 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
INFO - 2024-01-30 10:14:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-30 10:14:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-30 10:14:15 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bcpet_db`.`ambulatoir`, CONSTRAINT `FK_ambulatoir_pet_profile` FOREIGN KEY (`pet_id`) REFERENCES `pet_profile` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION) in D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 D:\Project (external)\BCPetHouse\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `am...', 0)
#1 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `am...')
#2 D:\Project (external)\BCPetHouse\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `am...')
#3 D:\Project (external)\BCPetHouse\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `am...', Array, false)
#4 D:\Project (external)\BCPetHouse\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Project (external)\BCPetHouse\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 D:\Project (external)\BCPetHouse\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\Project (external)\BCPetHouse\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 D:\Project (external)\BCPetHouse\app\Controllers\Ambulatoir.php(186): CodeIgniter\BaseModel->save(Array, true)
#9 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
#10 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
#11 D:\Project (external)\BCPetHouse\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\Project (external)\BCPetHouse\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 D:\Project (external)\BCPetHouse\system\Commands\Server\rewrite.php(47): require_once('D:\\Project (ext...')
#14 {main}
CRITICAL - 2024-01-30 10:14:15 --> Cannot add or update a child row: a foreign key constraint fails (`bcpet_db`.`ambulatoir`, CONSTRAINT `FK_ambulatoir_pet_profile` FOREIGN KEY (`pet_id`) REFERENCES `pet_profile` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ambulatoir` (`date_checkup`, `amnesa`, `status_present`, `clinical_finding`, `medication`, `created_at`, `updated_at`) VALUES (:date_checkup:, :amnesa:, :status_present:, :clinical_finding:, :medication:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Ambulatoir.php(186): CodeIgniter\BaseModel->save([...], true)
 7 SYSTEMPATH\CodeIgniter.php(915): App\Controllers\Ambulatoir->save()
 8 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ambulatoir))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Project (external)\\BCPetHouse\\public\\index.php')
